test/parallel/test-whatwg-url-searchparams-constructor.js

```diff
@@ -154,7 +154,7 @@ test(function() {
 }, "Constructor with sequence of sequences of strings");
 
 [
-//   { "input": {"+": "%C2"}, "output": [[" ", "\uFFFD"]], "name": "object with +" },
+  { "input": {"+": "%C2"}, "output": [["+", "%C2"]], "name": "object with +" },
```

Comment:
can you update the Refs SHA? This fix was in [`e94c604`](https://github.com/w3c/web-platform-tests/commit/e94c6049160a9b8dc8b4c47c50c313b025f7232b)